Addon.Episerver.EnvironmentSynchronizer/Configuration/ConfigurationReader.cs

```diff
@@ -12,25 +12,31 @@ public class ConfigurationReader : IConfigurationReader
 	{
 		private static readonly ILogger Logger = LogManager.GetLogger();
 
+		private readonly ISynchronizerConfiguration configuration;
```

Comment:
private variables should be prefixed with _ aka _configuration. Sorry - missed this in the initial review.

Reply:
Hi, just updated.